### DISCUSSION TITLE
RTL and LTR modifiers

### DISCUSSION THREAD

**Discussion by ma3ain** [cosscom/coss#4] [Suggestions]
Congratulations on the launch of your new product OriginUI! It’s exciting to see your hard work come to fruition, and I can’t wait to explore all the features.

I wanted to suggest considering the addition of right-to-left support for Arabic language users from the beginning. This feature would greatly enhance accessibility and usability for a broader audience. [RTL support](https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support)
 
Thank you for your continuous efforts, and I look forward to seeing how your product evolves!

**Comment by pasqualevitiello** [comment]
Added RTL support: https://github.com/origin-space/originui/pull/3
